backend/app/routes/admin/getAdminList.js

```diff
@@ -0,0 +1,34 @@
+const Admin = require('../../models/Admin');
+
+function getPropsFromAdminData(a) {
+  let json = '{';
+  if (!a.includes('_id')) json += '"_id":0,';
+  a.forEach(e => json += `"${e}": "$${e}",`);
+  json = json.replace(/.$/, '}')
+  return JSON.parse(json);
+}
+
+module.exports = async (req, res, next) => {
+  const options = req.query;
+  const propsFromAdminData = ['email', 'firstName'];
+  const limitOfPagination = 3;
+  let { pageNumber, isSuperAdmin } = options;
+  const pipeline = [
+    { $project: getPropsFromAdminData(propsFromAdminData) },

```

Comment:
I dont understand the need for getPropsFromAdminData function. I dont think so you need it